commons-dlp/src/main/java/org/exoplatform/commons/dlp/service/impl/DlpPositiveItemServiceImpl.java

```diff
@@ -95,17 +95,19 @@ public Long getDlpPositiveItemsCount() {
 
   private DlpPositiveItem fillDlpPositiveItemFromEntity(DlpPositiveItemEntity dlpPositiveItemEntity) throws Exception {
     DlpPositiveItem dlpPositiveItem = new DlpPositiveItem();
+    DlpServiceConnector dlpServiceConnector = (DlpServiceConnector) dlpOperationProcessor.getConnectors().get(dlpPositiveItemEntity.getType());
     dlpPositiveItem.setId(dlpPositiveItemEntity.getId());
     dlpPositiveItem.setType(dlpPositiveItemEntity.getType());
     dlpPositiveItem.setKeywords(dlpPositiveItemEntity.getKeywords());
-    dlpPositiveItem.setAuthor(dlpPositiveItemEntity.getAuthor());
-    dlpPositiveItem.setAuthorDisplayName(organizationService.getUserHandler()
-                                                            .findUserByName(dlpPositiveItemEntity.getAuthor())
-                                                            .getDisplayName());
+    if(dlpPositiveItemEntity.getAuthor() != null && !dlpPositiveItemEntity.getAuthor().isEmpty()) {
+      dlpPositiveItem.setAuthor(dlpPositiveItemEntity.getAuthor());
+      dlpPositiveItem.setAuthorDisplayName(organizationService.getUserHandler()
+                                                              .findUserByName(dlpPositiveItemEntity.getAuthor())
+                                                              .getDisplayName());
+      dlpPositiveItem.setIsExternal(dlpServiceConnector.checkExternal(dlpPositiveItemEntity.getAuthor()));
+    }
```

Comment:
If author is __system, due to old datas, 
organizationService.getUserHandler().findUserByName(dlpPositiveItemEntity.getAuthor()).getDisplayName()); 
will throw NPE.

So I think we should do that : 
```suggestion
   
      dlpPositiveItem.setAuthor(dlpPositiveItemEntity.getAuthor());
      User author = organizationService.getUserHandler()
                                                              .findUserByName(dlpPositiveItemEntity.getAuthor();
      if (user!=null) {
         dlpPositiveItem.setAuthorDisplayName(user.getDisplayName());
         dlpPositiveItem.setIsExternal(dlpServiceConnector.checkExternal(dlpPositiveItemEntity.getAuthor()));
    }
```

Reply:
done !